roles/kubernetes/secrets/files/make-ssl.sh

```diff
@@ -105,14 +105,13 @@ fi
 if [ -n "$HOSTS" ]; then
     for host in $HOSTS; do
         cn="${host%%.*}"
-        gen_key_and_cert "node-${host}" "/CN=system:node:${cn}/O=system:nodes"
+        gen_key_and_cert "node-${host}" "/CN=system:node:${cn,,}/O=system:nodes"
```

Comment:
neat, never seen that syntax before
```
${parameter^pattern}
${parameter^^pattern}
${parameter,pattern}
${parameter,,pattern}

    This expansion modifies the case of alphabetic characters in parameter. The pattern is expanded 
to produce a pattern just as in filename expansion. Each character in the expanded value of parameter 
is tested against pattern, and, if it matches the pattern, its case is converted. The pattern should not 
attempt to match more than one character. The ‘^’ operator converts lowercase letters matching pattern 
to uppercase; the ‘,’ operator converts matching uppercase letters to lowercase. The ‘^^’ and ‘,,’ expansions
convert each matched character in the expanded value; the ‘^’ and ‘,’ expansions match and convert only 
the first character in the expanded value. If pattern is omitted, it is treated like a ‘?’, which matches every 
character. If parameter is ‘@’ or ‘*’, the case modification operation is applied to each positional parameter 
in turn, and the expansion is the resultant list. If parameter is an array variable subscripted with ‘@’ or ‘*’, 
the case modification operation is applied to each member of the array in turn, and the expansion is the 
resultant list.
```
https://www.gnu.org/software/bash/manual/bashref.html#Shell-Parameter-Expansion

Reply:
It's a dirty hack but it works.